stream-product/product-core/src/main/java/com/backbase/stream/product/mapping/ProductMapper.java

```diff
@@ -215,6 +216,13 @@ default CurrentInvestment mapCurrentInvestment(BigDecimal bigDecimal) {
         return new CurrentInvestment().amount(bigDecimal);
     }
 
+    default BigDecimal mapCurrentInvestment(CurrentInvestment currentInvestment) {
```

Comment:
why do we need this guy here?

Reply:
InvestmentAccount has field of type CurrentInvestment. In order to map it to AccountArrangementItemPost, we need to add a proper mapping. Without it there is a compilation error:
`java: Can't map property "CurrentInvestment currentInvestment" to "BigDecimal currentInvestmentValue". Consider to declare/implement a mapping method: "BigDecimal map(CurrentInvestment value)".`